cea/interfaces/dashboard/inputs/routes.py

```diff
@@ -146,4 +165,10 @@ def route_table_post(db):
     else:
         cea.utilities.dbf.dataframe_to_dbf(table_df, location)
 
-    return render_template('table.html', pk='Name', table_name=db, table_columns=db_info['fieldnames'])
\ No newline at end of file
+    return render_template('table.html', pk='Name', table_name=db, table_columns=db_info['fieldnames'])
+
+
+def df_to_json(file_location):
+    table_df = geopandas.GeoDataFrame.from_file(file_location)
+    table_df = table_df.to_crs(epsg=4326)  # make sure that the geojson is coded in latitude / longitude
```

Comment:
`4326` should probably be a constant somewhere. hmm... [googled it](https://gis.stackexchange.com/questions/3334/difference-between-wgs84-and-epsg4326) it seems this is just a name for the reference system we use. maybe we could find other places that use this?

Reply:
```suggestion
from cea.utilities.standardize_coordinates import get_geographic_coordinate_system
    table_df = table_df.to_crs(get_geographic_coordinate_system())  # make sure that the geojson is coded in latitude / longitude
```
we could change it to the current method that we have now, since they sort of mean the same thing.